android/src/main/java/com/swmansion/gesturehandler/core/GestureHandlerOrchestrator.kt

```diff
@@ -702,5 +709,8 @@ class GestureHandlerOrchestrator(
       state == GestureHandler.STATE_CANCELLED ||
         state == GestureHandler.STATE_FAILED ||
         state == GestureHandler.STATE_END
+
+    private fun isActive(state: Int) =
+      state == GestureHandler.STATE_ACTIVE
```

Comment:
I don't think this is necessary, it's used once while in every other place, a simple equality is used.

Reply:
removed in [b4fe4d8](https://github.com/software-mansion/react-native-gesture-handler/pull/3095/commits/b4fe4d841142d1505350266337a659a1aa5c3d45)